async-opcua-client/src/session/services/subscriptions/event_loop.rs

```diff
@@ -30,6 +30,9 @@ pub(crate) struct SubscriptionEventLoop {
     // This is true if the client has received BadTooManyPublishRequests
     // and is waiting for a response before making further requests.
     is_waiting_for_response: bool,
+    // This is true if the client has received a no_subscriptions response,
+    // and is waiting for a manual trigger or successful response before resuming publishing.
+    is_paused: bool,
```

Comment:
Maybe something like this:

```suggestion
    waiting_for_response: bool,
    // This is true if the client has received a no_subscriptions response and is waiting
    // for a manual trigger or successful response before resuming publishing.
    no_active_subscription: bool,
```

_Note that I updated `is_waiting_for_response` on purpose as well_